benchmark/reformat_input.py

```diff
@@ -337,19 +339,192 @@ def reformat_treepuzzle(gene_tree,
     msa_fa.write(output_msa_phy_fp, format='phylip')
 
 
+def _merge_genbank_seqs(genbank_fp):
+    """ Merge one to multiple sequences in a GenBank file into one.
+
+    Parameters
+    ----------
+    genbank_fp: string
+        file path to genome in GenBank format
+
+    Returns
+    -------
+    tuple of (
+        skbio.Sequence,
+            Genome sequence and metadata
+        dict of { list of [ string, int, int, string ] }
+            Gene name : translation, start, end, and strand
+    )
+    """
+    loci = []
```

Comment:
I once did the way you (and another guy) recommended. However, I noticed a significant performance loss. On July 17 I wrote the following statement in an email to Jenya: _I realized that for a multi-sequence GenBank file, scikit-bio cannot parse all sequences together, but has to do this one per time (the developer can specify a sequence index). In my script I followed this mechanism to loop through all sequences. However, I found that the processing speed is substantially slower than splitting the master genbank files into multiple child files with one sequence each, and feed scikit-bio with them sequentially. For example for a 9-sequence file, the former took 18 seconds to parse while the later only took 4 seconds._